zipkin-storage/cassandra-v1/src/main/resources/cassandra-schema-cql3-upgrade-1.txt

```diff
@@ -28,3 +28,7 @@ ALTER TABLE zipkin.service_names
 ALTER TABLE zipkin.traces
     WITH compaction = {'class': 'org.apache.cassandra.db.compaction.DateTieredCompactionStrategy', 'max_window_size_seconds': '86400'}
     AND default_time_to_live =  604800;
+
+ALTER TABLE zipkin.tags
```

Comment:
I don't think this is needed since this is a brand new table that already has a default ttl, no?

Reply:
I wasn't aware of the reason why this one exists. I will remove it.

Reply:
Remvoed